types/qrcode-svg/qrcode-svg-tests.ts

```diff
@@ -0,0 +1,24 @@
+import * as QRCode from "qrcode-svg";
```

Comment:
`import QRCode = require("qrcode-svg");` because QRCode is a class

Reply:
Done